content/android/_index.md

```diff
@@ -11,4 +11,8 @@ listdescription: ""
 
```

Comment:
I agree. The first option makes the most sense to me. Thanks for ensuring that the site structure and URLs are sensible.